spec/e2e/scope_spec.rb

```diff
@@ -97,6 +97,26 @@
     end
   end
 
+  describe 'Person.scope :great_students, -> (identifier, score) { where("#{identifier}.score > ?", score)' do
+    before(:each) do
+      Person.scope :great_students, ->(identifier, score) { where("#{identifier}.score > ?", score || 41) }
```

Comment:
This is fine, though I think that in 8.0 we support calling `identity` from within a scope now